microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,233 @@
+      "start_url": "/",
+      "short_name": "PWAmp",
+      "description": "A skinable music player app to play your favorite mp3 files",
+      "display": "standalone",
+      "icons": [
+        {
+          "src": "./favicon-256.png",
+          "sizes": "256x256"
+        }
+      ],
+      "edge_side_panel": {}
+    }
+    ```
+
+
+<!-- ====================================================================== -->
+## Adapt your app to the minimum sidebar width
+
+The sidebar in Microsoft Edge has a default minimum width of 376 pixels and can be resized by users. Therefore, your app's layout should support the 376 pixels minimum width and be responsive.
+
+While building your app, you can test if the app's layout is usable when narrow, and responsive, by using the **Device Emulation** tool in Microsoft Edge DevTools. To learn more, see [Verify that the webpage layout is usable when narrow](../../devtools-guide-chromium/accessibility/narrow.md).
+
+If your app's layout can't support the 376 pixels minimum width, you can define your preferred width by using the `preferred_width` property in your web app manifest. For example:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "standalone",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+When you define a preferred width in your app's manifest, the following happens:
+
+* When your app is opened in the sidebar, the sidebar is automatically resized to your preferred width.
+* Users can still resize the sidebar to make it larger than your preferred width, but not smaller.
+
+
+<!-- ====================================================================== -->
+## Build a sidebar-only app
+
+One of the most important benefits of building an app as a PWA is that, from one code base, your app adapts to all devices and operating systems, whatever their capabilities and screen sizes.
+
+By using the `edge_side_panel` manifest member, you can also make your app installable as a standalone app, or pinned to the sidebar in Microsoft Edge.
+
+However, if you prefer to build an app that can only be pinned to the sidebar but not installed as a standalone app, omit the `display` member from your web app manifest, or set its value to `browser`:
+
+```json
+{
+  "name": "PWAmp music player",
+  "lang": "en-US",
+  "start_url": "/",
+  "short_name": "PWAmp",
+  "description": "A skinable music player app to play your favorite mp3 files",
+  "display": "browser",
+  "icons": [
+    {
+      "src": "./favicon-256.png",
+      "sizes": "256x256"
+    }
+  ],
+  "edge_side_panel": {
+    "preferred_width": 480
+  }
+}
+```
+
+
+<!-- ====================================================================== -->
+## Detect when your app runs in the sidebar
```

Comment:
This could also be for sites.

Reply:
Not sure what to do here. This documentation (and the entire parent area) is focused on PWA apps only. So rephrasing to "Detect when your app or site runs in the sidebar" feels a bit out of place here.

Reply:
Let's use "Detect usage in the sidebar" then

Reply:
Detect usage in the sidebar